### ISSUE TITLE
rt1 - Data types in current file not pretty printed

### ISSUE THREAD

**Issue by francisdb** [unisonweb/unison#353]
```
  ┌
  │  134 | > [Textual "test"]
  │          ⧩
  │          [Con #"2pPZ 1 "test"]
  └
```
I’d like to get some more useful output instead of this Con #"2pPZ 1

> Paul Chiusano @pchiusano 16:51
> @francisdb there’s no typeclass for that, but all values are showable and should print nicely - it looks like the data types in the current file just aren’t being added to the prettyprint environment, which is a bug. You can submit a ticket for it. :)

**Comment by aryairani** [comment]
@pchiusano I wanna know exactly where that open quote in `#”2pPZ` is coming from first/too.

**Comment by francisdb** [comment]
@aryairani the output has changed

```
  │  210 | > Textual "test"
  │          ⧩
  │          #2pPZi4FuaB#1 "test"
  └
  ┌
  │  211 | > Err "t"
  │          ⧩
  │          #2pPZi4FuaB#4 "t"
  └
```

**Comment by aryairani** [comment]
Fixed in 53310f78.
